### PR TITLE
Data release 2021_11_16_01_01_32; Update stat var group proto

### PR DESCRIPTION
**Pull request by shifucun**
Autopush can not be deployed due to additional proto fields. Updated the proto and data version.

### CODE REVIEW

File: test/integration/golden_response/get_statvar_group_node/school_race.json [hunk 1]
```diff
@@ -1,59 +1,86 @@
 {
   "absoluteName": "Person With Enrollment Level = Enrolled in College Undergraduate Years, Race",
-  "childStatVarGroups": [
+  "childStatVars": [
```
Comment (shifucun) [inline test/integration/golden_response/get_statvar_group_node/school_race.json:3]: @pradh This seems to be a big change. Can you confirm?

Comment (pradh) [reply test/integration/golden_response/get_statvar_group_node/school_race.json:3]: This is because of cl/410075652

**Review by pradh — APPROVED** [review]
Sorry for the delay!

**Comment by shifucun** [comment]
thanks!

**Comment by shifucun** [comment]
thanks!

**Pull request merged**